<?xml version="1.0"?>
<launch>

    <!-- 加载参数文件 -->
    <rosparam command="load" file="$(find d2p)/config/detector_params.yaml" ns="/target_detector"/>

    <!-- 设置日志级别 -->
    <env name="ROSCONSOLE_CONFIG_FILE" value="$(find d2p)/config/rosconsole.conf"/>

    <!-- 启动目标检测节点 -->
    <node pkg="d2p" type="target_detector" name="target_detector" output="screen">
        <!-- 设置日志级别 -->
        <env name="ROSCONSOLE_MIN_SEVERITY" value="DEBUG"/>
    </node>


</launch>
